apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -295,8 +307,20 @@ class ShareAllowedDialog extends React.Component {
                       onClick={select}
                       readOnly
                       value={this.props.shareUrl}
-                      style={{cursor: 'copy', width: 500}}
+                      style={{cursor: 'copy', width: 450}}
                     />
+                    <button
+                      type="button"
+                      id="share-dialog-copy-button"
+                      style={
+                        this.state.hasBeenCopied
+                          ? styles.copyButtonLight
+                          : styles.copyButton
+                      }
+                      onClick={wrapShareClick(this.copy.bind(this), 'copy')}
```

Comment:
do we need to bind `this.copy`? it's an arrow function, so it should be bound to `this` automatically

Reply:
Good call.  Updated this and all the existing instances of this syntax in the file where possible.